### PR TITLE
EBNT fixed use_local_installation changing cwd

### PR DESCRIPTION
**Pull request by ElmiRakh**

### CODE REVIEW

File: src/ebonite/build/builder/base.py [hunk 1]
```diff
@@ -93,10 +93,12 @@ def _write_requirements(self, target_dir):
             requirements = self.provider.get_requirements()
             logger.debug('Auto-determined requirements for model: %s.', requirements.to_pip())
             if not ebonite_from_pip():
+                cwd = os.getcwd()
                 from setup import setup_args  # FIXME only for development
                 requirements += list(setup_args['install_requires'])
                 logger.debug('Adding Ebonite requirements as local installation is employed...')
                 logger.debug('Overall requirements for model: %s.', requirements.to_pip())
+                os.chdir(cwd)
```
Comment (mike0sv) [inline src/ebonite/build/builder/base.py:101]: add try/finally jsut to be sure...

File: src/ebonite/runtime/client/base.py [hunk 2]
```diff
@@ -93,7 +93,7 @@ def __call__(self, *args, **kwargs):
 
 
 def _bootstrap_method(method: InterfaceMethodDescriptor):
-    logger.debug(f'Bootstraping server method "%s" with %s argument(s)...', method.name, len(method.args))
+    logger.debug(f'Bootstraping server method "{method.name}" with {len(method.args)} argument(s)...')
```
Comment (mike0sv) [inline src/ebonite/runtime/client/base.py:96]: this was on purpose, this way you avoid calling `str()` if you do not log because of the log level

Comment (ElmiRakh) [reply src/ebonite/runtime/client/base.py:96]: So noqa this line, then?

Comment (ElmiRakh) [reply src/ebonite/runtime/client/base.py:96]: Fixed

**Pull request merged**